docs/pages/machine-id/reference/cli.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: Machine ID CLI Reference
+description: CLI reference for Teleport Machine ID.
+---
+
+# Machine ID CLI Reference
+
+## `tbot start`
+
+Starts the Machine ID client `tbot`, fetching and writing certificates to disk
+at a set interval.
+
+<Details scope={["cloud"]} scopeOnly={true}>
+```code
+$ tbot start \
+   --data-dir=/var/lib/teleport/bot \
```

Comment:
Maybe a nitpick, but this is the default value for `--data-dir` so this doesn't strictly need to be specified.